docs/USING_PRO.md

```diff
@@ -99,30 +110,34 @@ The tokenizer defines how to turn markdown text into tokens.
 // Create reference instance
 const marked = require('marked');
 
-// Get reference
-const tokenizer = new marked.Tokenizer();
-const originalCodespan = tokenizer.codespan;
 // Override function
-tokenizer.codespan = function(src) {
-  const match = src.match(/\$+([^\$\n]+?)\$+/);
-  if (match) {
-    return {
-      type: 'codespan',
-      raw: match[0],
-      text: match[1].trim()
-    };
+const tokenizer = {
+  codespan(src) {
+    const match = src.match(/\$+([^\$\n]+?)\$+/);
+    if (match) {
+      return {
+        type: 'codespan',
+        raw: match[0],
+        text: match[1].trim()
+      };
+    }
+
+    // return false to use original codespan tokenizer
+    return false;
```

Comment:
What is the difference between returning `false` vs `undefined`?

Reply:
`undefined` is a valid return for the tokenizer basically saying skip this token for the current `src`. I think `false` is a good thing to check because it is a way for the extension author to explicitly let marked know it wants it to fall back to the previous function instead of skipping the token.

We could change it to `true` or something else but I don't think it should always fall back with `undefined`.

Reply:
That answers my question, thanks!